netstandard/ref/netstandard.csproj

```diff
@@ -9,7 +9,10 @@
     <NuGetTargetMoniker>.NETStandard,Version=v1.7</NuGetTargetMoniker>
     <IsCoreAssembly>true</IsCoreAssembly>
     <!-- disable obsolete warnings/errors -->
-    <NoWarn>$(NoWarn);0618;0619;0809</NoWarn>
+    <NoWarn>$(NoWarn);0618;0619;0809</NoWarn>  
+
+    <DefaultBuildAllTarget>Build</DefaultBuildAllTarget>
```

Comment:
Im not sure what happened with the NoWarning thing, but the build all targets properties were necessary to stop the clash logger from reporting clashing.